internal/api/v1beta1/group.go

```diff
@@ -252,6 +327,28 @@ func (h Handler) AddGroupUsers(ctx context.Context, request *frontierv1beta1.Add
 
 func (h Handler) RemoveGroupUser(ctx context.Context, request *frontierv1beta1.RemoveGroupUserRequest) (*frontierv1beta1.RemoveGroupUserResponse, error) {
 	logger := grpczap.Extract(ctx)
+	_, err := h.orgService.Get(ctx, request.GetOrgId())
+	if err != nil {
+		logger.Error(err.Error())
+		switch {
+		case errors.Is(err, organization.ErrDisabled):
+			return nil, grpcOrgDisabledErr
+		case errors.Is(err, organization.ErrNotExist):
+			return nil, grpcOrgNotFoundErr
+		default:
+			return nil, grpcInternalServerError
+		}
+	}
+
+	owners, err := h.userService.ListByGroup(ctx, request.GetId(), schema.OwnerRelationName)
+	if err != nil {
+		logger.Error(err.Error())
+		return nil, grpcInternalServerError
+	}
+	if len(owners) == 1 {
```

Comment:
Hey where do we check the user we are trying to remove is part of owners list? This would fail the call if I am just removing a random user as well right if we have 1 owner?